### DISCUSSION TITLE
Context Menu / Menu Flyout not working

### DISCUSSION THREAD

**Discussion by CosminCAS123** [AvaloniaUI/Avalonia#15942] [Q&A] [answered]
![image](https://github.com/AvaloniaUI/Avalonia/assets/79158479/2877a223-fd2c-4386-9e0e-9fa0ed4f40af)
I mean, it works, but I have to right-click multiple times before the menu appears. Sometimes even only 1 time, but it's pretty random. Also it looks like it only works on when I right-click the text only, the icon or the rest of the space not really working.
![image](https://github.com/AvaloniaUI/Avalonia/assets/79158479/a234dc78-05dd-474e-af86-3eb41300dede)

**Comment by IanRawley** [comment, marked as answer]
Try setting a background on the StackPanel, something like transparent.  Without a background it's not hit enabled, which is probably why your clicks aren't being picked up.
